Documentation/Getting-Started/glossary.md

```diff
@@ -0,0 +1,135 @@
+
+### Krew Plugin
+
+The [Rook Krew plugin](../Troubleshooting/krew-plugin.md) is a tool to help troubleshoot your Rook cluster.
+
+### OpenShift
+
+[OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform) Container Platform is a distribution of the Kubernetes container platform.
+
+### Object Bucket Claim (OBC)
+
+An Object Bucket Claim (OBC) is custom resource which requests a bucket (new or existing) from a Ceph object store. For further reference please refer to [OBC Custom Resource](../Storage-Configuration/Object-Storage-RGW/ceph-object-bucket-claim.md).
+
+### Object Bucket (OB)
+
+An Object Bucket (OB) is a custom resource automatically generated when a bucket is provisioned. It is a global resource, typically not visible to non-admin users, and contains information specific to the bucket.
+
+### Toolbox
+
+The [Rook toolbox](../Troubleshooting/ceph-toolbox.md) is a container with common tools used for rook debugging and testing.
+
+## Rook CRDs
+### CephCluster CRD
+
+CephCluster CRD is used by Rook to allow creation and customization of storage clusters through the custom resource definitions (CRDs). For more information about the different modes of creation of clusters, refer to our [documentation](../CRDs/Cluster/ceph-cluster-crd.md).
+
+### External Storage Cluster
+
+An [external cluster](../CRDs/Cluster/external-cluster.md) is a Ceph configuration that is managed outside of the local K8s cluster.
+
+### Host Storage Cluster
+
+A [host storage cluster](../CRDs/Cluster/host-cluster.md) is where Rook configures Ceph to store data directly on the host devices.
+
+### PVC Storage Cluster
+
+In a [PersistentVolumeClaim-based cluster](../CRDs/Cluster/pvc-cluster.md), the Ceph persistent data is stored on volumes requested from a storage class of your choice.
+
+### Stretch Storage Cluster
+
+A stretched cluster is a deployment model in which two or more virtualization host servers are part of the same logical cluster but are located in separate geographical locations. To support this scenario, Ceph has integrated support for [stretch clusters](../CRDs/Cluster/stretch-cluster.md).
+
+### CephBlockPool CRD
+
+CephBlockPool CRD is used by Rook to allow creation and customization of storage pools. For more information and examples refer to this [documentation](../CRDs/Block-Storage/ceph-block-pool-crd.md).
+
+### CephBlockPoolRados Namespace CRD
+
+CephBlockPoolRados Namespace CRD is used by Rook to allow creation of Ceph BlockPool RadosNamespaces. For more information and examples refer to this [documentation](../CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md).
+
+### CephRBDMirror CRD
+
+CephRBDMirror CRD is used by Rook to allow creation and updating rbd-mirror daemon(s) through the custom resource definitions (CRDs). For more information and examples refer to this [documentation](../CRDs/Block-Storage/ceph-rbd-mirror-crd.md).
+
+### CephFilesystem CRD
```

Comment:
Need to alphabetize all the CRDs. For example, this should be before the rbd mirror.